transport/http2_client.go

```diff
@@ -98,7 +98,8 @@ type http2Client struct {
 
 	initialWindowSize int32
 
-	bdpEst *bdpEstimator
+	bdpEst          *bdpEstimator
+	outQuotaVersion uint64
```

Comment:
No, don't.  There should only be one version per connection, not one per stream-quota.